openhtf/plugs/__init__.py

```diff
@@ -482,7 +482,11 @@ def initialize_plugs(self, plug_types=None):
           # Now the instance has its own copy of the test logger.
           plug_instance.logger = self._logger
       except Exception:  # pylint: disable=broad-except
-        _LOG.error('Exception instantiating plug type %s', plug_type)
+        _LOG.error('Exception instantiating plug type %s', plug_type,
+                   exc_info=True)
+        # Also log to the test record.
+        self._logger.error('Exception instantiating plug type %s', plug_type,
+                           exc_info=True)
```

Comment:
Confirmed, this logger does have console logging (dependent on command line flags), so logging once is better.